Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -57,17 +53,6 @@ struct UnifiedTestRunner {
     /// strings indicating cases to skip. If the array contains a single string "*" all tests in the file will be
     /// skipped.
     func runFiles(_ files: [UnifiedTestFile], skipTests: [String: [String]] = [:]) throws {
```

Comment:
I highly recommend reviewing this with "hide whitespace changes" on, as most of the diff here is from indenting a bunch of this code to wrap it in a try/catch.